frontend/src/components/Following/FollowingButton.js

```diff
@@ -0,0 +1,67 @@
+import '../../config';
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
+
+// Stylesheet
+import './FollowingButton.scss'
+
+export default function FollowingButton(props)
+{
+    const [isFollowing, setFollowing] = useState(false);
+    const [errorMsg, setErrorMsg] = useState("");
+
+    const follow = (event) => {
+        event.preventDefault();
+        axios.post(global.config.BACKEND_URL + "/account/following/newFollower/" + props.username)
+            .then(
```

Comment:
I know I've been trying to put simple code on one line but I'm not sure what you think about it. I would put `.then(setFollowing(true))` on one line and then `.catch()` on the next